### DISCUSSION TITLE
Case-Insensitive Search?

### DISCUSSION THREAD

**Discussion by bcrotty** [eza-community/eza#867] [Q&A, Howto, Help] [answered]
Is there a way to make your search pattern case-insensitive?

Thanks!

**Comment by gierens** [comment, marked as answer]
Can you elaborate what you mean by search pattern?

In case you mean glob patterns this has nothing to do with eza and more with your shell:

**ZSH:**
1. you can make glob patterns case-insensitive in general by unsetting `CASE_GLOB`:

![Screenshot from 2024-05-23 09-36-38](https://github.com/eza-community/eza/assets/49617392/e3823789-a83f-4f62-a3c7-ee964c20d2b4)

2. or you can set `extendedglob` and make use of the `(#i)` to mark a glob as case-insensitive:

![Screenshot from 2024-05-23 09-38-30](https://github.com/eza-community/eza/assets/49617392/f9c13f49-6f92-4624-a939-1acc09fbbbb0)

**BASH:**
Here you can set `nocaseglob`:

![Screenshot from 2024-05-23 09-40-15](https://github.com/eza-community/eza/assets/49617392/7d63fea5-8e68-4f2b-9b9c-76d563fcfd3f)

**Comment by bcrotty** [comment]
That's helpful. Thanks!
